### ISSUE TITLE
Feature : Add Save Button or Saved Status Indicator to API Key Modal

### ISSUE THREAD

**Issue by blackkspydo** [TGlide/codeverter#11]
# Problem Statement

The website currently provides a modal with an input field for users to paste their API key. Although the API key is saved to the local storage upon pasting, there is no clear indication or feedback for the user to know if the key has been saved successfully. This lack of confirmation can lead to confusion and uncertainty for users, potentially diminishing their experience with the platform.

# Proposed Solution

I propose adding a "Save" button or a "Saved" status indicator within the API key modal to provide clear feedback to users when their API key has been saved successfully. When users paste their API key into the input field and click the "Save" button, the application should save the key to the local storage and display a "Saved" status indicator (e.g., a checkmark icon or a brief text message) to confirm that the action has been completed.

**Comment by TGlide** [comment]
Good idea! I think a debounced saved indicator would suffice. Maybe a small text beneath the input that shows up for a bit.
